app/dashboard/ethelo.py

```diff
@@ -0,0 +1,54 @@
+from grants.models import GrantQuerySet, Grant
+
+
+EXPORT_FILENAME = "grants_export_for_ethelo.json"
+
+
+def get_grants_from_database(start_grant_number: int, end_grant_number: int=None) -> dict:
```

Comment:
@thelostone-mc 

this probably should have a test written, but i'm honestly not sure how to create a few grants such that the database query would return a few deterministic grant objects.

i read in the code that the [local environment should create "Initial test data"](https://docs.gitcoin.co/mk_setup/), but i didn't see any in my localhost:8000 site. maybe it's only created during tests?